CHANGES.rst

```diff
@@ -1,6 +1,9 @@
 Changes in 0.8.1
 ----------------
 - Added Django 1.10 support `198 <https://github.com/django-money/django-money/issues/198>`_ (`Dmitry Dygalo <https://github.com/Stranger6667>`_)
+- Changed auto conversion of currencies using djmoney_rates (added in 0.7.3) to
```

Comment:
With this change in mind, would you mind bumping the version for all the above unreleased changes to 0.9 (changing the subheading)

The version itself is bumped in setup.py